js/utils.js

```diff
@@ -97,8 +97,11 @@ Fliplet.Registry.set('dynamicListUtils', (function() {
             break;
         }
 
-        var fileIDs = files.map(function(fileUrl) {
-          var matchedFileUrl = fileUrl.match(/v1\/media\/files\/([0-9]+)/);
+        var fileIDs = files.map(function(file) {
+          var url = typeof file === 'string'
+            ? file
+            : file.url;
+          var matchedFileUrl = url.match(/v1\/media\/files\/([0-9]+)/);
```

Comment:
Can we not use this?

```suggestion
          return Fliplet.Media.getIdFromRemoteUrl(url);
```

Reply:
@tonytlwu fixed